test/smoke/MLH-584_ageFieldSpinner_TCA.js

```diff
@@ -0,0 +1,24 @@
+const sel = require('../../data/selectors.json');
+
+describe('MLH-584 field age tests to check spinners', function () {
+
+    it('TCA-004 Verify first click spinner Up.', function () {
+        browser.url('');
+        $$(sel.spinner)[0].click();
+        const Num1 = $(sel.age).getValue();
+        expect(Num1).toEqual("1");
+    });
+
+    it('TCA-005 Verify that spinner Up is present.', function () {
```

Comment:
Test doesn't pass but spinner is present. Try adding moveTo() to point mouse over the age field first and then checking for spinner to present

Reply:
Still doesn't go through. Test shows there's no spinner displayed.